flit_core/flit_core/config.py

```diff
@@ -201,18 +231,18 @@ def _check_glob_patterns(pats, clude):
                 .format(clude, p)
             )
 
-        normp = osp.normpath(p)
+        normp = posix_normpath(PurePosixPath(p))
```

Comment:
This looks like a behaviour change on Windows -- since it's switching from Windows paths to POSIX paths.

Reply:
The path handling seemed buggy/inconsistent here since os.path.normpath is filesystem/os specific and these validations I think need to be consistent across platforms.

Reply:
The tests already expect these paths to always be posix style at the moment, as this path is not an absolute path on windows:
https://github.com/pypa/flit/blob/6982faee6f69e21327bdc8cf245fe99d57ea9fc3/flit_core/flit_core/tests/test_config.py#L109